lib/response.ts

```diff
@@ -383,3 +383,19 @@ export class IssueProxyResponse {
     readonly redirectUri?: string
   ) { }
 }
+
+export class TxMessageWithDetailResponse {
+  constructor(
+    readonly msgIndex: number,
+    readonly requestType: string,
+    readonly details: any
+  ) { }
+}
+
+export class TxMessageListResponse {
+  constructor(
+    readonly messages: Array<TxMessageWithDetailResponse>,
+    readonly prePageToken: string = "",
+    readonly nextPageToken: string= "",
```

Comment:
chore: could you do align the code style?

Reply:
?? please let me know in more details.

Reply:
Fixed at aedf661ed352d69f35e7f4c43536a731c9b7ea6b